packages/webview_flutter/webview_flutter/lib/src/v4/src/webview_controller.dart

```diff
@@ -0,0 +1,263 @@
+  /// `/Users/username/Documents/www/index.html`.
+  ///
+  /// Throws a `PlatformException` if the [absoluteFilePath] does not exist.
+  Future<void> loadFile(String absoluteFilePath) {
+    return platform.loadFile(absoluteFilePath);
+  }
+
+  /// Loads the Flutter asset specified in the pubspec.yaml file.
+  ///
+  /// Throws a `PlatformException` if [key] is not part of the specified assets
+  /// in the pubspec.yaml file.
+  Future<void> loadFlutterAsset(String key) {
+    assert(key.isNotEmpty);
+    return platform.loadFlutterAsset(key);
+  }
+
+  /// Loads the supplied HTML string.
+  ///
+  /// The [baseUrl] parameter is used when resolving relative URLs within the
+  /// HTML string.
+  Future<void> loadHtmlString(String html, {String? baseUrl}) {
+    assert(html.isNotEmpty);
+    return platform.loadHtmlString(html, baseUrl: baseUrl);
+  }
+
+  /// Makes a specific HTTP request ands loads the response in the webview.
+  ///
+  /// [method] must be one of the supported HTTP methods in [LoadRequestMethod].
+  ///
+  /// If [headers] is not empty, its key-value pairs will be added as the
+  /// headers for the request.
+  ///
+  /// If [body] is not null, it will be added as the body for the request.
+  ///
+  /// Throws an ArgumentError if [uri] has an empty scheme.
+  Future<void> loadRequest(
+    Uri uri, {
+    LoadRequestMethod method = LoadRequestMethod.get,
+    Map<String, String> headers = const <String, String>{},
+    Uint8List? body,
+  }) {
+    if (uri.scheme.isEmpty) {
+      throw ArgumentError('Missing scheme in uri: $uri');
+    }
+    return platform.loadRequest(LoadRequestParams(
+      uri: uri,
+      method: method,
+      headers: headers,
+      body: body,
+    ));
+  }
+
+  /// Accessor to the current URL that the WebView is displaying.
+  ///
+  /// If no URL was ever loaded, returns `null`.
+  Future<String?> currentUrl() {
+    return platform.currentUrl();
+  }
+
+  /// Checks whether there's a back history item.
+  Future<bool> canGoBack() {
+    return platform.canGoBack();
+  }
+
+  /// Checks whether there's a forward history item.
+  Future<bool> canGoForward() {
+    return platform.canGoForward();
+  }
+
+  /// Goes back in the history of this WebView.
+  ///
+  /// If there is no back history item this is a no-op.
+  Future<void> goBack() {
+    return platform.goBack();
+  }
+
+  /// Goes forward in the history of this WebView.
+  ///
+  /// If there is no forward history item this is a no-op.
+  Future<void> goForward() {
+    return platform.goForward();
+  }
+
+  /// Reloads the current URL.
+  Future<void> reload() {
+    return platform.reload();
+  }
+
+  /// Clears all caches used by the WebView.
+  ///
+  /// The following caches are cleared:
+  ///	1. Browser HTTP Cache.
+  ///	2. [Cache API](https://developers.google.com/web/fundamentals/instant-and-offline/web-storage/cache-api) caches.
+  ///    These are not yet supported in iOS WkWebView. Service workers tend to
```

Comment:
Referencing `WkWebView` is probably too much of an internal implementation detail for a client-facing API.